### PR TITLE
fix:[CDS-89531]: Update rollback deployment doc with a limitation

### PR DESCRIPTION
**Pull request by krishi0408**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/continuous-delivery/manage-deployments/rollback-deployments.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ Post deployment rollback initiates a rollback of your most recent successful dep
 * Rollback steps can use expressions that refer to steps executed during the previous pipeline execution. The expressions are automatically resolved with values from the original execution.
 * The stages should roll back in reverse order when  deployment rollback is triggered.
 * Rolling back to a previous post-production deployment is only possible for executions that occurred within the past 30 days.
+* If the failure strategy marks an execution as successful and ignored the failures, it doesn't mean you can roll back that execution.
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/manage-deployments/rollback-deployments.md:28]:
```suggestion
* You cannot roll back an execution if was marked successful by the ignore failures failure strategy.
```

Comment (roshnisarangadharan) [reply docs/continuous-delivery/manage-deployments/rollback-deployments.md:28]: Your statement sounds more casual. Please see if the suggestion is technically accurate.

**Review by roshnisarangadharan — APPROVED** [review]
Approving with the assumption that you will consider my suggestion.

File: docs/continuous-delivery/manage-deployments/rollback-deployments.md (hunk 1 above)
Comment (krishi0408) [reply docs/continuous-delivery/manage-deployments/rollback-deployments.md:28]: Hey @roshnisarangadharan we need to mention that if we use action mark as success.

Comment (krishi0408) [reply docs/continuous-delivery/manage-deployments/rollback-deployments.md:28]: Yes @roshnisarangadharan look good

**Review by roshnisarangadharan — APPROVED** [review]

**Pull request merged**